app/jobs/reports/sp_user_counts_report.rb

```diff
@@ -15,7 +15,24 @@ def perform(_date)
       user_counts = transaction_with_timeout do
         Db::Identity::SpUserCounts.call
       end
+
+      track_report_data_events(user_counts)
       save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
     end
+
+    private
+
+    def track_report_data_events(user_counts)
+      user_counts.each do |hash|
+        track_report_data_event(
+          Analytics::REPORT_USER_COUNTS,
+          issuer: hash['issuer'],
+          user_total: hash['total'],
+          ial1_user_total: hash['ial1_total'],
+          ial2_user_total: hash['ial2_total'],
+          app_id: hash['app_id'].to_s,
+        )
+      end
+    end
```

Comment:
do we also want to add an event for the total across all SPs?

Reply:
yes

Reply:
i'm going to do 3.  total number of users registered to sps,  total number of users fully registered, and total number of users

Reply:
ok, are you going to do all 3 in this PR or in other PRs?

Reply:
scratch that...just doing the one for now...can do the others in a separate pr